src/__tests__/resources/timeSeriesList.unit.spec.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 Cognite AS
+import MockAdapter from 'axios-mock-adapter';
+import { uniqBy } from 'lodash';
+import { CogniteClient } from '../..';
+import { AssetList } from '../../resources/classes/assetList';
+import { TimeSeriesList } from '../../resources/classes/timeSeriesList';
+import { randomInt, setupLoggedInClient } from '../testUtils';
+
+describe('TimeSeriesList class unit test', async () => {
+  let axiosMock: MockAdapter;
+  let client: CogniteClient;
+  let newAsset: any;
+  let createdTimeSeries: TimeSeriesList;
+  let timeSeriesWithAssetId: any;
+  let createdAssets: AssetList;
+  let datapointArray: any[];
+  let assetsArray: any[] = [];
+  let timeseriesArray: any[] = [];
+  beforeAll(() => {
+    client = setupLoggedInClient();
+    axiosMock = new MockAdapter(client.instance);
+  });
+  beforeEach(async () => {
+    axiosMock.reset();
+
+    // Create assets
+    assetsArray = [];
+    for (let index = 0; index < 5; index++) {
+      newAsset = {
+        name: 'test-asset' + randomInt(),
+        externalId: 'asset' + randomInt(),
+        id: randomInt(),
+      };
+      assetsArray.push(newAsset);
+    }
+    axiosMock
+      .onPost(new RegExp('/assets$'), { items: assetsArray })
+      .replyOnce(200, { items: assetsArray });
+    createdAssets = await client.assets.create(assetsArray);
```

Comment:
Why do you create assets?

Reply:
So I can add `assetId` to a timeseries, and test `getAsset()` method. Do you have any suggestions how to do this without creating them?